ddtrace/internal/schema/__init__.py

```diff
@@ -18,10 +20,13 @@ def _validate_schema(version):
 
 
 __schema_version = os.getenv("DD_TRACE_SPAN_ATTRIBUTE_SCHEMA", default="v0")
+__remove_client_service_names = asbool(os.getenv("DD_TRACE_REMOVE_CLIENT_SERVICE_NAMES_ENABLED", default=False))
```

Comment:
the name of the env has been changed to `DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED`

Reply:
done